lib/rules/best-practises/no-unused-import.js

```diff
@@ -17,6 +37,11 @@ const meta = {
   schema: null,
 }
 
+// meant to skip Identifiers that are part of the ImportDirective that defines a name
+function isImportIdentifier(node) {
+  return !node.parent || node.parent.type === 'ImportDirective'
```

Comment:
Just as a FYI: the `?.` optional chaining operator is now supported by all the actively maintained versions of node, so maybe you can start leveraging it:

```js
return node.parent?.type === 'ImportDirective'
```

Reply:
Thanks! although `node.parent?.type === 'ImportDirective'` is not equivalent to the current implementation, and we'd have to update eslint's configuration to support a newer parser.  #26 it's very little work but still out of scope for this pr

Reply:
> `node.parent?.type === 'ImportDirective'` is not equivalent to the current implementation

I'll stop reviewing pull requests after 10 PM :man_facepalming: